x-pack/legacy/plugins/graph/public/components/graph_visualization/_graph_visualization.scss

```diff
@@ -0,0 +1,60 @@
+.gphEdge {
+  fill: $euiColorMediumShade;
+  stroke: $euiColorMediumShade;
+  stroke-width: 2;
+  stroke-opacity: 0.5;
+
+  &:hover {
+    stroke-opacity: 0.95;
+    cursor: pointer;
+  }
+
+  &--selected {
+    stroke: $euiColorDarkShade;
+    stroke-opacity: 0.95;
+  }
+
+  &--inferred {
+    stroke-dasharray: 5, 5;
+  }
+}
+
+.gphNode__label {
+  @include gphSvgText;
+  cursor: pointer;
+  &--html {
+    @include euiTextTruncate;
+    text-align: center;
+  }
+}
+
+.gphNode__markerCircle {
+  fill: $euiColorDarkShade;
+  stroke: $euiColorEmptyShade;
+}
+
+.gphNode__markerText {
+  @include gphSvgText;
+  font-size: $euiSizeS - 2px;
+  fill: $euiColorEmptyShade;
+}
+
+.gphNode__circle {
+  fill: $euiColorMediumShade;
+  &--selected {
+    stroke-width: $euiSizeXS;
+    stroke: transparentize($euiColorPrimary, 0.25);
+  }
+}
+
+.gphNode__text {
+  fill: $euiColorDarkestShade;
```

Comment:
These fills need to be always black or always white, not switchable via theme. So that when going from 

![image](https://user-images.githubusercontent.com/549577/65911656-f10adb00-e39a-11e9-9fb1-c7d56a06ad14.png)


**to dark mode, you get**

![image](https://user-images.githubusercontent.com/549577/65911784-244d6a00-e39b-11e9-8c17-beca77b5f169.png)

**Instead of**

![image](https://user-images.githubusercontent.com/549577/65912066-98880d80-e39b-11e9-99fd-858f1cc8d22c.png)


```suggestion
  fill: $euiColorInk;
```

Reply:
Good point, fixed. As the visualization now supports dark colors, I removed the light color filter from the default palette.